docs/adding-a-page.md

```diff
@@ -1,337 +0,0 @@
-# addons-frontend
-
-This will outline what is required to add a page to the project. A basic knowledge of
-[react](https://facebook.github.io/react/docs/getting-started.html) and
-[redux](http://redux.js.org/) is assumed.
-
-**Note:** This page is very out-of-date and does not reflect our practices anymore. We now use [redux-saga](https://github.com/redux-saga/redux-saga) for API requests. See `amo/components/Categories.js` for a more modern example of a component that makes API/async requests for data.
```

Comment:
Rather than removing this page entirely, we could leave this section and mention these docs do need to be expanded. Pointing someone to a modern component with sagas + `shallowUntilTarget` tests is still better than saying nothing. I think code makes for lousy documentation but if we direct people to right bits of code it still beats just saying nothing.

Reply:
Indeed!

Reply:
Yeah, this section is problematic but I don't know if you should spent time to rewrite it yet. As matt mentioned, maybe just link to a new-ish component as an example.